### PR TITLE
convert band agreement page into blocks, for overriding in downstream plugins

### PR DESCRIPTION
**Pull request by binary1230**
- turn parts of the band agreement into blocks for extending via plugins
- requires venue INI settings PR

allow other plugins (like magstock) to override sections of agreement without needing to paste the entire agreement.html here.

now includes all of @cybersenshi's updated text (but not styling), it appears to work great.  if this is what we go for, take the rest of @cybersenshi's stuff from https://github.com/magfest/magstock/pull/64 and put it in here.

an example of a PR that overrides is here: https://github.com/magfest/magstock/pull/65

downstream plugins can also turn off sections by blanking them out, for instance if we didn't want to have a merch section, you can do: ```{% block band_text_merch %}{% endblock %}```

### CODE REVIEW

**Comment by kitsuta** [comment]
+1, good use of Django templating. Much nicer than a hundred includes.

**Comment by binary1230** [comment]
meetig with chaney, TODO: this is the new form stuff, needs to go back to the old form stuff

File: bands/templates/bands/agreement-base.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    <p>
+        By submitting your band's information<span class="printhide"> below</span>, you agree to the above terms.
+    </p>
+{% endblock %}
+
+{% block band_information_form %}
+<h3>
+    Your Information
+</h3>
+<form method="post" action="agreement" class="form-horizontal" role="form">
```
Comment (binary1230) [inline bands/templates/bands/agreement-base.html]: @cybersenshi notes that this form here is actually the new styling and we need to put back in the old styling for this PR

**Comment by binary1230** [comment]
OK, this form now has the original styling back in, so should be good to go.  Looks good locally. @cybersenshi plz give this a review and when you +1, it's ready to merge.

After, merge #28

**Comment by cybersenshi** [comment]
+1

**Comment by EliAndrewC** [comment]
+1 (belatedly)

**Pull request merged**